macros/edr/data_monitoring/monitors_query/column_monitoring_query.sql

```diff
@@ -174,6 +168,70 @@
 
 {% endmacro %}
 
+{% macro get_column_metrics_unpivot_query(column_metrics, column_obj, metric_name_to_type, timestamp_column, dimensions) %}
```

Comment:
i assume that you replaced it with this in some version and either changed it back or reverted the change, so either
1. remove this
2. change the code before to use this